imports/ui/containers/Preset/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import MedContainer from '../../components/MediumContainer';
```

Comment:
Component 'containers' for the purposes of layout seems like an anti-pattern.
It could be semantically unclear that this is a Layout container vs. a 'Container Component', which is really just a class that wraps a pure functional component. In general avoid naming your components 'BlahContainer' when importing. 

If this is code (line 2) imports a React component used to control the layout of your app, then use a more semantic name related to what the component does, to avoid confusion between 'Container Components' and your app's Layout components.